docs/rst/introduction/tools/fastddsspy.rst

```diff
@@ -0,0 +1,28 @@
+.. _vulcanexus_fastddsspy:
+
+Fast DDS Spy
+============
+
+*Fast DDS Spy* is CLI interactive tool that allows to introspect a ROS 2 network in human readable format.
+It is possible to query the network about the Participants connected, their endpoints (Publishers and Subscriptions) and the topics they communicate in.
+It is also possible to see the user data sent through network topics in a schematic format in run time.
+
+Overview
+--------
+
+*Fast DDS Spy* is a tool that introspect or "sniffing" DDS packages in the network and maintain a local database that is accessible from a interactive CLI.
+*Fast DDS Spy* responds to user commands introduced by text and print in ``stdin`` the information requested.
+This tool has several commands to interact with, that allows to get information regarding the status of the network.
+It supports to list topics existing, list Participants, Publishers, Subscriptions and even read user data in real time in a human readable format.
```

Comment:
I think support + gerund is the correct way of writing this, but I'm not 100% positive.